test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -119,11 +119,13 @@ public void checkPermission(Permission perm, Object context)
 
         private final InputStream input;
         private final T out;
+        private boolean autoCloseOut;
 
-        private StreamGobbler(InputStream input, T out)
+        private StreamGobbler(InputStream input, T out, boolean autoCloseOut)
```

Comment:
Is this one of the two bugs mentioned on the ticket?

Reply:
Yes, the process would just spin waiting for more input, it was a bug in `ToolRunner`